spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/StreamParser.java

```diff
@@ -154,18 +154,16 @@ private StreamNode eatStream() {
 				bridge = true;
 			}
 		}
-
 		Tokens tokens = getTokens();
-		List<ModuleNode> moduleNodes;
+		List<ModuleNode> moduleNodes = new ArrayList<>();
 		if (bridge) {
 			// Create a bridge module to hang the source/sink channels off
 			tokens.decrementPosition(); // Rewind so we can nicely eat the sink channel
-			moduleNodes = new ArrayList<>();
-			moduleNodes.add(new ModuleNode(null, "bridge", tokens.peek().startPos,
+			moduleNodes.add(new ModuleNode(null, "noop", tokens.peek().startPos,
```

Comment:
Since `noop` processor is already available, I ended up adding it here instead of `bridge` processor. We could still add `bridge` processor in SCSM if we really need or may be rename `noop` to `bridge` and use `bridge` processor.